src/components/Signup.js

```diff
@@ -69,6 +70,7 @@ export default class Signup extends Component {
       .then((response)=> {
         if (response.status >= 200 && response.status < 300) {
           this.setState({ isLoaded: true });
+          ReactPixel.track('CompleteRegistration', {email: this.state.emailValue});

```

Comment:
It seems to me that this line will return an exception if `process.env.REACT_APP_FACEBOOK_PIXEL` will be empty